src/solver/optimisation/opt_appel_solveur_lineaire.cpp

```diff
@@ -241,7 +241,14 @@ static SimplexResult OPT_TryToCallSimplex(
     if (options.useOrtools)
     {
         const bool keepBasis = (optimizationNumber == PREMIERE_OPTIMISATION);
-        solver = ORTOOLS_Simplexe(&Probleme, solver, keepBasis);
+        solver = ORTOOLS_Simplexe(&Probleme,
```

Comment:
this is getting long, we could create an appropriate struct for ortools/solver options

Reply:
OptimizationOptions?

Reply:
I had the same idea, without reading your comment. See these [2 consecutive commits](https://github.com/AntaresSimulatorTeam/Antares_Simulator/pull/1837/files/07074583e5d29d8c8beb869d3b1f8ee070741511..23f67dc1d69f2d65bff801ecd544bfbe9ab47a25)